docs/pages/blog/introducing-the-row-grouping-feature.md

```diff
@@ -1,48 +1,50 @@
 ---
-title: 'Introducing the Row Grouping feature'
-description: After more than a year of development on the DataGrid, the first premium feature is live.
+title: 'Customize your data grid with row grouping'
```

Comment:
I'm not sure about "Customize" this feels too generic. Maybe something in this order?

```suggestion
title: 'Analyze your data with the data grid row grouping'
```

Reply:
I would say "Let user organize their data with row grouping" but it does not sound sexy enough for a title

Another idea would be "Empower users with the new row grouping feature" or "Row grouping is live" and modify the description to provide more information about what is row grouping

Reply:
hmm in that case, maybe we swap the headline and subheader so they read something like:

# Give your users more freedom with data grid row grouping
## The new row grouping feature gives your users more customization options for organizing their data.

Reply:
Nit/question: what is the right way of writing Data Grid (capitalized) or data grid (all lower)?

Reply:
That's actually a great question @danilo-leal and worth clarifying. The original post used `DataGrid`, but elsewhere on the site and throughout the docs it appears that components are most commonly written all in lowercase, with spaces between words—`data grid`.

Reply:
+1 for data grid when a reference to the component and DataGrid when a reference to the JavaScript module, in headers, instead of Data grid, we have used Data Grid so far.

Reply:
To continue with the discussion opened in the survey PR, what about Data Grid Pro? Is it DataGridPro, data grid pro, or something else? What's the convention?

Reply:
> DataGridPro when we talk about what developers import in their code.

Do you mean `import DataGridPro from '@mui/x-data-grid-pro'`?
If yes, then it should be Data Grid and Data Grid Pro,  shouldn't it? I believe we don't need to account for JS's case sensitiveness to refer to the component in common paragraphs...

Reply:
Sorry, I think that I could have made my previous comments clearer. I find them confusing reading them back.

## 1. What I believe we have done so far

### 1.a. When we talk about the React component

When we talk about the **exported JavaScript module**. I believe that the standard is to use the Uppercase without spaces and a backquote. Examples:

  - `Masonry` https://mui.com/components/masonry/#columns
  <img width="273" alt="Screenshot 2022-02-19 at 16 28 13" src="https://user-images.githubusercontent.com/3165635/154807403-013005a9-db5a-4fdb-a2a3-df62234e54c8.png">

  - `ButtonGroup` https://mui.com/components/button-group/#split-button
  <img width="277" alt="Screenshot 2022-02-19 at 16 37 04" src="https://user-images.githubusercontent.com/3165635/154807726-ea9cd832-1d4d-43dd-9ecb-e075ec140dae.png">

  - Same with the GitHub labels: https://github.com/mui/material-ui/labels?q=grid

  However, I recall having a discussion with @mbrookes about this very problem. In some places, we diverge. We don't use the backquotes. It makes reading the text a bit easier (especially with repetitions), it's fewer interruptions of the normal reading flows. But at the same time, the backquotes makes everything clearer, e.g. SpeedDialActions https://mui.com/components/speed-dial/#persistent-action-tooltips
  <img width="317" alt="Screenshot 2022-02-19 at 16 26 39" src="https://user-images.githubusercontent.com/3165635/154807314-6c162685-4b2a-4ccc-a4bb-04545c9beedb.png">

  <img width="273" alt="Screenshot 2022-02-19 at 16 28 13" src="https://user-images.githubusercontent.com/3165635/154807403-013005a9-db5a-4fdb-a2a3-df62234e54c8.png">

### 1.b. When we talk about the **component in general**.

When it's about the general notion, I believe that using the lower case with spaces. Examples:

  - autocomplete https://mui.com/components/autocomplete/#highlights
    <img width="286" alt="Screenshot 2022-02-19 at 16 24 50" src="https://user-images.githubusercontent.com/3165635/154807238-4a1e097d-147b-46f5-8d09-641bbfd2f9b1.png">

  - date picker https://mui.com/components/date-picker/#static-mode
    <img width="341" alt="Screenshot 2022-02-19 at 16 33 46" src="https://user-images.githubusercontent.com/3165635/154807614-f35f5e65-6c12-4a6c-93c4-577e70b0c11e.png">
   - Same with the GitHub labels: https://github.com/mui/material-ui/labels?q=grid
 
 Of course, if it's used at the beginning of a sentence or beginning a header, we capitalize the first, letter, e.g. 
 
  - App bar https://material.io/components/app-bars-top
  <img width="407" alt="Screenshot 2022-02-19 at 16 41 57" src="https://user-images.githubusercontent.com/3165635/154807875-d3a8fb55-93b3-41e3-977b-f9675bcab884.png">

  - Button group https://mui.com/components/button-group/
  <img width="296" alt="Screenshot 2022-02-19 at 16 45 44" src="https://user-images.githubusercontent.com/3165635/154808010-bcd9721a-c56c-4338-ae75-a13caac68358.png">

## 2. What should we do going forward?

In practice, the above is partially enforced. You can find many instances where we don't follow this target. But my point is more, here is what I believe we have been aiming for. So, this asks to solve one question: How do we normalize? How do we fix the existing inconsistencies? Do we want to continue in the current direction or do we want to course correct?

Reply:
I see a few different ways to approach this, but I think the current convention works. We just need to be diligent about enforcing it.

I'm going to restate what I understand here, to make sure we're on the same page:

- when referring a MUI component, write it like a React component with backticks - `DataGridPro`
- when referring to the general concept of a component, write it in normal sentence case & spacing - _a floating action button_

I'm sure there are valid exceptions to these rules (we may not want to use backticks in headlines, for example), but this sets the simplest and clearest baseline.

Reply:
Ok, so we seem good to go then! I'll review the survey post with this convention in mind and look for places to standardize it across the docs in further PRs.

Reply:
There is a potential issue here that I just discovered. When I use the Chrome screen reader extension, it just skips over text inside of `code backticks`, making the sentence incomprehensible. According to [Google's documentation accessibility guides](https://developers.google.com/style/accessibility), we may encounter the same issue if we use camelCaseNames in our docs.

In the interest of accessibility, then, we should avoid `DataGridPro` in favor of Data Grid Pro.

Reply:
But then that would mean that using backticks, in general, is a bad practice? There should be a way to optimize backticks...

Reply:
That does seem to be the case, yes. Is there a way to tell screen readers to ignore the backticks? Or is that too much trouble when the alternative is just to leave them out?

Reply:
When I use the screen reader here on GitHub, it reads `DataGridPro` with no trouble as though it was written Data Grid Pro. When I tried it on a Notion page it skipped over the text.

Reply:
I wonder if @eps1lon has any insight to drop here. To summarize: it seems that text written wrapped in backticks is not being read by screen readers.

Reply:
I have tried on VoiceOver, it doesn't announce/read the `<code>`.

Reply:
Started this page https://www.notion.so/mui-org/Accessibility-4c152039577e4372ad4adf2ccb4ed67b

Reply:
@samuelsycamore Is there a place where we should document this or simply updating all the docs to be in sync would be enough? Then the team can copy.